### DISCUSSION TITLE
Hardware Scaling

### DISCUSSION THREAD

**Discussion by Daedalus3** [OpenSlides/OpenSlides#6279] [Q&A] [answered]
Hi folks,

first of all: thanks for making this awesome piece of software. 

I have a question regarding hardware scaling, as I can't find anything in the docs (which might be my fault).

-What hardware is needed for e.g. hosting a conference with around 500 participants? 
-Does the hardware requirement change when there is quite some voting with multiple votings at the same time?
-Is the big mode still needed with version 3 (I can only find references to it in docs regarding older versions?

Any help would be highly appreciated. Thanks!

**Comment by tsiegleauq** [comment, marked as answer]
Nothing too fancy I suppose, but we can really only give advice due to our experience and none of our recommendations would be compulsory to give you a satisfactory result.

To build the client you will need at least 8gb of RAM. The processing power of your machine might be somewhat neglectable for normal conferences, the connection quality and fast network IO would be the usual braking point.

> Does the hardware requirement change when there is quite some voting with multiple votings at the same time?

yes, a lot. processing the votes will take some time and power.

> Is the big mode still needed with version 3 (I can only find references to it in docs regarding older versions?

With 3.4 docker is needed, big mode does not exist anymore afaik

@gsiv for more information
